packages/liveblocks-node/src/client.ts

```diff
@@ -1851,6 +1851,41 @@ export class Liveblocks {
     return this.getRoomSubscriptionSettings(params, options);
   }
 
+  /**
+   * Returns the room subscription settings for a user.
+   * @param params.userId The user ID to get the room subscription settings from.
+   * @param options.signal (optional) An abort signal to cancel the request.
+   */
+  public async getUserRoomSubscriptionSettings(
+    params: {
+      userId: string;
+    },
+    options?: RequestOptions
+  ): Promise<
+    Page<
+      {
+        roomId: string;
+      } & RoomSubscriptionSettings
+    >
```

Comment:
I don't think we need to export it (unless someone asks) but we could make this a named type `UserRoomSubscriptionSettings` like `ThreadParticipants`